dbflute-engine/src/main/java/org/dbflute/logic/doc/historyhtml/DfSchemaHistory.java

```diff
@@ -205,10 +205,20 @@ protected void assertDiffElementMap(String key, Object value) {
     public void loadHacoMap() {
         String clientDirPath = ".";
         DfHacoMapFile hacoMapFile = new DfHacoMapFile(() -> DBFluteSystem.currentLocalDateTime());
-        // TODO hakiba add exception handling  (2018/04/28)
-        List<DfHacoMapPiece> pieceList = hacoMapFile.readPieceList(clientDirPath);
-        OptionalThing<DfHacoMapPickup> optPickup = hacoMapFile.readPickup(clientDirPath);
-        _hacoMapPickup = hacoMapFile.merge(optPickup, pieceList);
+        // TODO done hakiba add exception handling  (2018/04/28)
+        try {
+            List<DfHacoMapPiece> pieceList = hacoMapFile.readPieceList(clientDirPath);
+            OptionalThing<DfHacoMapPickup> optPickup = hacoMapFile.readPickup(clientDirPath);
+            _hacoMapPickup = hacoMapFile.merge(optPickup, pieceList);
+        } catch (RuntimeException e){
```

Comment:
What is this exception handling for?
If you want to improve debug information,
e.g. you should show "clientDIrPath" in the exception message.

While, the addItem("Exception") is unneeded because IllegalStateException wraps it.
(Sometimes it can do for emphasis, but I think it does not mean here)

Reply:
Since clientDirPath is a fixed value, I decided that log output is not necessary. However, FilePath may be changed, so I will log it.